httphandler.go

```diff
@@ -25,6 +29,22 @@ type SMSRequest struct {
 
 const smsCharLength = 160
 
+const helpDeskTemplate = `
+<html><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8"></head>
+<body>
+<pre style="font-family: arial; font-size: 15px">					
+Name: %v
+Email: %v
+Phone number:%v
+Call me back: %v
+Browser: %v
+Browser version: %v
```

Comment:
It might be useful to the helpdesk to know on which server the issue was reported (maybe also include the URL here, e.g. `https://stellenbosch.imqs.co.za/`)